docs/index.md

```diff
@@ -28,6 +28,233 @@ Core utilities such as Tracer, Logger, Metrics, and Event Handler will be availa
+      name = "iam_for_lambda"
+      assume_role_policy = <<EOF
+        {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Action": "sts:AssumeRole",
+              "Principal": {
+                "Service": "lambda.amazonaws.com"
+              },
+              "Effect": "Allow"
+            }
+          ]
+        }
+        EOF
+	}
+    resource "aws_lambda_function" "test_lambda" {
+      filename      = "lambda_function_payload.zip"
+      function_name = "lambda_function_name"
+      role          = aws_iam_role.iam_for_lambda.arn
+      handler       = "index.test"
+      runtime 		= "nodejs16.x"
+      layers 		= ["arn:aws:lambda:{region}:094274105915:layer:AWSLambdaPowertoolsTypeScript:1"]
+      source_code_hash = filebase64sha256("lambda_function_payload.zip")
+    }
+    ```
+
+=== "Amplify"
+
+    ```zsh
+    # Create a new one with the layer
+    ❯ amplify add function
+    ? Select which capability you want to add: Lambda function (serverless function)
+    ? Provide an AWS Lambda function name: <NAME-OF-FUNCTION>
+    ? Choose the runtime that you want to use: NodeJS
+    ? Do you want to configure advanced settings? Yes
+    ...
+    ? Do you want to enable Lambda layers for this function? Yes
+    ? Enter up to 5 existing Lambda layer ARNs (comma-separated): arn:aws:lambda:eu-central-1:094274105915:layer:AWSLambdaPowertoolsTypeScript:1
+    ❯ amplify push -y
+    # Updating an existing function and add the layer
+    ❯ amplify update function
+    ? Select the Lambda function you want to update test2
+    General information
+    - Name: <NAME-OF-FUNCTION>
+    ? Which setting do you want to update? Lambda layers configuration
+    ? Do you want to enable Lambda layers for this function? Yes
+    ? Enter up to 5 existing Lambda layer ARNs (comma-separated): arn:aws:lambda:eu-central-1:094274105915:layer:AWSLambdaPowertoolsTypeScript:1
+    ? Do you want to edit the local lambda function now? No
+    ```
+
+=== "Get the Layer .zip contents"
+	Change `{region}` to your AWS region, e.g. `eu-west-1`
+
+    ```bash title="AWS CLI"
+	aws lambda get-layer-version-by-arn --arn arn:aws:lambda:{region}:094274105915:layer:AWSLambdaPowertoolsTypeScript:1 --region {region}
+    ```
+
+    The pre-signed URL to download this Lambda Layer will be within `Location` key.
+
+???+ warning "Warning: Limitations"
+
+	Container Image deployment (OCI) or inline Lambda functions do not support Lambda Layers.
+
+If you use `esbuild` to bundle your code, make sure to exclude `@aws-lambda-powertools`   from being bundled since the packages will be brought by the Layer:
+
+=== "SAM"
+
+    ```yaml hl_lines="5"
+    MyLambdaFunction:
+        Type: AWS::Serverless::Function
+        Properties:
+            ...
+            Metadata: 
+              # Manage esbuild properties
+              BuildMethod: esbuild
+              BuildProperties:
+                Minify: true
+                External:s
```

Comment:
Is this `s` in `External:s` (<- this) on purpose? If yes please resolve this comment.

Reply:
oups , well catched